### PR TITLE
Remove homework gem

### PR DESCRIPTION
**Pull request by s-espinosa**
@AliSchlereth Let me know if there were additional changes you were thinking of making to these exercises. Thanks!

### CODE REVIEW

File: core-types/ints_and_floats_test.rb [hunk 1]
```diff
@@ -2,24 +2,23 @@
 gem 'minitest', '~> 5.2'
 require 'minitest/autorun'
 require 'minitest/pride'
-require 'minitest/homework'
-
-class FixnumTest < Minitest::Homework
-  be_gentle!
 
+class FixnumTest < Minitest::Test
```
Comment (AliSchlereth) [inline core-types/ints_and_floats_test.rb]: Should we change the class name too? Looks like you renamed the file already, correct?

File: core-types/ints_and_floats_test.rb [hunk 2]
```diff
@@ -2,24 +2,23 @@
...
   def test_add
     assert_equal __, 6.send(:+, 3)
```
Comment (AliSchlereth) [inline core-types/ints_and_floats_test.rb]: I'm not super stoked about showing them send here. I think it mostly would be confusing at this point. Thoughts?

**Comment by AliSchlereth** [comment]
I might go back and add some more functionality to the Fixnum Core Type later, but go ahead and merge now in case I don't have time this weekend.  I'm also thinking it might be good to have a hash playground that deals more with nesting - but I  don't think that needs to be in the original Hash Core Type.
@s-espinosa

File: core-types/ints_and_floats_test.rb (hunk 1 above)
Comment (s-espinosa) [inline core-types/ints_and_floats_test.rb]: Yup!

File: core-types/ints_and_floats_test.rb (hunk 2 above)
Comment (s-espinosa) [inline core-types/ints_and_floats_test.rb]: Agreed.

**Pull request merged**